mapping/sustainability.yaml

```
@@ -785,12 +829,14 @@
         environmental
         gender empowerment
         \[add free text to explain the co-benefits\]
-    mapping: 'Project level: For each impact identified add a `benefit` object to the `benefits` array and assign it a locally unique `id`. Set the `title` as the list code and add details explaining the benefit to `.description`.'
+    mapping: |-
+        Project level: For each impact identified add a `benefit` object to the `benefits` array.
+
+        - Set the `title` as the list code and add details explaining the benefit to `.description`.
```

Copy edit. Capitalise definition. Remove unnecessary list formatting.

```suggestion
        Project level: For each co-benefit, add a `Benefit` object to the `benefits` array, map the option from the list to its `.title` and map the explanation to its `.description`.
```